### PR TITLE
Update explicit-fields-the-val-keyword.md

### PR DESCRIPTION
**Pull request by V0d01ey**

### CODE REVIEW

**Comment by LisandroSu** [comment]
Здравствуйте, @V0d01ey . Ваш запрос еще находится на рассмотрении. Как только будет принято решение, мы обязательно сообщим Вам.  
  
 С уважением, 
Группа локализации Microsoft C&E

File: docs/fsharp/language-reference/members/explicit-fields-the-val-keyword.md [hunk 1]
```diff
@@ -24,18 +24,18 @@ val [ mutable ] [ access-modifier ] field-name : type-name
...
 - Тип, поддерживающий значение null как нормальное значение, как аномальное значение или как представление значения. Сюда входят классы, кортежи, записи, функции, интерфейсы, ссылочные типы .NET, тип `unit` и типы размеченного объединения.
 
-- Тип-значение .NET.
+- Тип значения .NET.
 
 - Структура, все поля которой поддерживают нулевое значение по умолчанию.
 
 
-Например, неизменяемое поле с именем `someField` имеет базовое поле в скомпилированном в .NET представлении с именем `someField@`, и вы обращаетесь к хранимому значению, используя свойство с именем `someField`.
+Например, неизменяемое поле с именем `someField` имеет резервное поле в скомпилированном в .NET представлении с именем `someField@`, и вы обращаетесь к хранимому значению, используя свойство с именем `someField`.
```
Comment (V0d01ey) [inline docs/fsharp/language-reference/members/explicit-fields-the-val-keyword.md]: "резервный" means "backup".
But "backing" here means more like "опорный".

**Comment by v-radelg** [comment]
@V0d01ey : Thank you for your feedback and suggestion! 

The feedback is considered. We will tune part of  your feedback in order to comply with Microsoft Style Guide, then release this article in the next loc release round.
		
Microsoft C&E Localization Team

**Comment by V0d01ey** [comment]
It's a pity that you ignored review messages.

**Comment by v-radelg** [comment]
Hello @V0d01ey  , thank for your message.

Our linguistic moderator reviewed all your changes and considered there were a few modifications they needed. 

Thank you for your support

Kind regards,  Microsoft C&E Localization Team

**Pull request merged**